jekyll-dev.yml

```
@@ -64,3 +64,8 @@ jekyll-generator-single-source:
   layout: 'page'
   multiple_products: false
   base_dest_path: 'docs'
+
+# Product name variables
+base_gateway: Kong Gateway
+mesh_product_name: Kong Mesh
```

```suggestion
mesh_product_name: Kuma
```
I think this should be Kuma here (and then it stays "Kong Mesh" in these files at the docs.konghq.com repo).